pkg/storage/named_preparer.go

```diff
@@ -0,0 +1,8 @@
+package storage
```

Comment:
Should this be in `sqlutils` instead of `storage`?

Reply:
Thanks Clay, I forgot to remove it from this package. Deleting it now